hack/update-godep-licenses.sh

```diff
@@ -75,6 +75,9 @@ process_content () {
     go4.org/*)
      package_root=$(echo ${package} |awk -F/ '{ print $1 }')
      ;;
+    gopkg.in/*)
```

Comment:
The LICENSES diff is impossible to read.  Can I get you to send a PR with *just* this change (and licenses updated), so I can verify that piece of it?

Reply:
Sent https://github.com/kubernetes/kubernetes/pull/56231